x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -480,10 +484,27 @@ export function App({
         // Temporarily required until the 'by value' paradigm is default.
         (dashboardFeatureFlag.allowByValueEmbeddables || Boolean(initialInput))
     ),
+    enableExportToCSV: Boolean(
+      lastKnownDoc?.state?.activeData && Object.keys(lastKnownDoc.state.activeData).length
+    ),
     isByValueMode: getIsByValueMode(),
     showCancel: Boolean(state.isLinkedToOriginatingApp),
     savingPermitted,
     actions: {
+      exportToCSV: () => {
+        const content = exportAsCSVs(
+          lastKnownDoc?.title || 'unsaved',
```

Comment:
needs i18n

Reply:
Maybe the i18n string can be centralized as well